magpie/utils.py

```diff
@@ -29,15 +32,16 @@
 
 CONTENT_TYPE_ANY = "*/*"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'str'](https://app.codacy.com/app/fmigneault/Magpie/pullRequest?prid=4650807)